cmd/minishift/cmd/start.go

```diff
@@ -274,11 +291,24 @@ func startHost(libMachineClient *libmachine.Client) (*host.Host, string) {
 		ShellProxyEnv:    shellProxyEnv,
 	}
 
-	fmt.Printf("Starting local OpenShift cluster using '%s' hypervisor...\n", machineConfig.VMDriver)
+	fmt.Printf(" using '%s' hypervisor ...\n", machineConfig.VMDriver)
 	var hostVm *host.Host
+
+	// configuration with these settings only happen on create
+	isRestart := cmdutil.VMExists(libMachineClient, constants.MachineName)
+	if !isRestart {
+		fmt.Println("-- Instance will be configured with ...")
+		fmt.Println("   Memory:   ", units.HumanSize(float64((machineConfig.Memory/units.KiB)*units.GB)))
+		fmt.Println("   vCPUs :   ", machineConfig.CPUs)
+		fmt.Println("   Disksize: ", units.HumanSize(float64(machineConfig.DiskSize*units.MB)))
+		fmt.Println("   Boot ISO: ", machineConfig.MinikubeISO)
```

Comment:
This might be  bit confusing, since the output is for example:

    Boot ISO:  https://github.com/minishift/minishift-b2d-iso/releases/download/v1.0.2/minishift-b2d.iso

This does not take into affect that the ISO is cached. Really I would expect something like:

    ~/.minishift/cache/iso/minishift-b2d.iso

where the ISO is cached. Given that we don't checksum the ISO and neither properly version, I am pretty sure we can even end up in cases where the cached ISO does not matched the displayed URL. 

I would suggest to remove this for now until we handle the ISO cache a bit better.